keel-echo/src/main/kotlin/com/netflix/spinnaker/keel/echo/ManualJudgementNotifier.kt

```diff
@@ -54,23 +60,27 @@ class ManualJudgementNotifier(
           "*${currentState.deliveryConfigName}* requires your manual approval for deployment " +
           "into the *${currentState.environmentName}* environment."
       ),
-      interactiveActions = EchoNotification.InteractiveActions(
-        callbackServiceId = "keel",
-        callbackMessageId = currentState.uid?.toString() ?: error("ConstraintState.uid not present"),
-        actions = listOf(
-          EchoNotification.ButtonAction(
-            name = "manual-judgement",
-            label = "Approve",
-            value = ConstraintStatus.OVERRIDE_PASS.name
+      interactiveActions = if (dynamicConfigService.isEnabled(INTERACTIVE_NOTIFICATIONS_ENABLED, false)) {
```

Comment:
I made this configurable and set to `false` by default so we can get this merged while we figure out what to do about the public API to handle Slack notification callbacks.

Reply:
Any reason you did this in a fast property vs a config value? For this kind of thing I would prefer controls in both, with a preference for the property to be read from a config value (because it's more similar to how we do other things right now).

Reply:
Sorry, I don't know what I was thinking... For some reason I thought `dynamicConfigService` defaulted to reading config from the config files. And I can't even say it was the first config property I added. Will fix! 🤦‍♂